pcdswidgets/eps_byteindicator.py

```diff
@@ -0,0 +1,155 @@
+from pydm.widgets import PyDMByteIndicator, PyDMChannel
+from qtpy.QtCore import Property, Qt
+from qtpy.QtGui import QColor
+from qtpy.QtWidgets import QTabWidget, QVBoxLayout, QWidget
+
+
+class ByteIndicator_NegativeNums(PyDMByteIndicator):
+    """
+        Modified Byte Indicator calss. Overides update_indicators
+            fucntion to allow for negative numbers to work for ByteIndicator
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def update_indicators(self):
+        if self._shift < 0:
+            value = int(self.value) << abs(self._shift)
+        else:
+            value = int(self.value) >> self._shift
+
+        bits = [(value >> i) & 1
+                for i in range(self._num_bits)]
+        for bit, indicator in zip(bits, self._indicators):
+            if self._connected:
+                if self._alarm_state == 3:
+                    c = self._invalid_color
+                else:
+                    c = self._on_color if bit else self._off_color
+            else:
+                c = self._disconnected_color
+            indicator.setColor(c)
+
+
+class EPSByteIndicator(QWidget):
+    """
+        Widget for displaying EPS interlocks
+    """
+
+    _qt_designer_ = {
+        "group": "PCDS Utilities",
+        "is_container": False,
+    }
+
+    template_widget = ByteIndicator_NegativeNums
```

Comment:
This seems like a typo, did you mean to set a type annotation?
```suggestion
    template_widget: ByteIndicator_NegativeNums
```